### PR TITLE
Use GitHub Actions, use pre-commit formatting

### LINKED ISSUE #52: Move CI to GH-Actions
Hi,

since this is the repository with OSS GH-Actions can be used for free. They are somewhat more responsive and industrial_ci and ros-tooling WG provide configurations for it. The benefits are additional checks, e.g., linters, uncrustify, etc.

I have have also a "standard" setup used across ros-control WG. Should I add this here?

### PR DESCRIPTION
**Pull request by livanov93**
* Update readme file

* rename repos file

* Initial configuration

* Add CI configuration for dockersim

Closes #52

### CODE REVIEW

**Comment by destogl** [comment]
PR is already huge. So I would like to merge this as it is and then address issues in separate PRs.

File: README.md [hunk 1]
```diff
@@ -71,5 +71,5 @@ after checking the current joint states (to create minimal increment for safety)
 joint. The commands are incremented in regards to /joint_states found when the node is run.
 
 ```
-ros2 run ur_ros2_control_demos test_driver 
+ros2 run ur_ros2_control_demos test_driver
```
Comment (AndyZe) [inline README.md:74]: I have one small request before merging:

Please add the basic instructions to get CI passing to the README here. Like, what clang format command to run?

Suggest to do it in this style:

```
## Contributors

Code should be formatted with clang-format 10 before submitting like this:  `clang_command`

Add any other tips here, too
```

**Comment by AndyZe** [comment]
Also, when I try to run pre-commit, this file is missing:

`=====> .pre-commit-config.yaml does not exist`

**Review by AndyZe — APPROVED** [review]
OK, I've tested this and pre-commit seems to work well. Let's get it merged and we can clean up minor details in followup PR's.

File: README.md (hunk 1 above)
Comment (AndyZe) [reply README.md:74]: I pushed a small commit to add this

**Pull request merged**